### PR TITLE
[fix] add custom textarea to have uniformity of textfield across rjsf

### PR DESCRIPTION
**Pull request by harkiratsm**
**Notes for Reviewers**

This PR fixes #

Added custom textarea widget to have a uniformity of textfield across rjsf

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryMeshInterface/helpers.js [hunk 1]
```diff
@@ -126,6 +127,12 @@ function jsonSchemaBuilder(schema, uiSchema) {
     return
   }
 
+  // This will removed, once the rjsf library issue is fixed for textarea
```
Comment (harkiratsm) [inline ui/components/MesheryMeshInterface/helpers.js:130]: This is yet another RJSF issue, I will open a issue. // @Abhishek-kumar09

Comment (harkiratsm) [reply ui/components/MesheryMeshInterface/helpers.js:130]: fixed

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
@harkiratsm Would you add a screenshot how it looks right now?

**Comment by harkiratsm** [comment]
Just an example  - 

image

**Comment by harkiratsm** [comment]
I guess the failing units test is related to https://github.com/meshery/meshery/pull/7634

**Pull request merged**